src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/Generated/ContainerServiceOperationsExtensions.cs

```diff
@@ -1,271 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@brendandburns I'm assuming this file was deleted for `ContainerServicesOperationsExtensions.cs`, but that file does not exist